pallets/post-follows/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = 'pallet-post-follows'
+version = '0.1.8'
+authors = ['DappForce <dappforce@pm.me>']
+edition = '2021'
+license = 'GPL-3.0-only'
+homepage = 'https://subsocial.network'
+repository = 'https://github.com/dappforce/subsocial-parachain'
+description = 'Pallet that allows to follow/unfollow posts'
+keywords = ['blockchain', 'cryptocurrency', 'social-network', 'news-feed', 'marketplace']
+categories = ['cryptography::cryptocurrencies']
+
+[features]
+default = ['std']
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks"]
+std = [
+    'codec/std',
+    'scale-info/std',
+    'frame-benchmarking/std',
+    'frame-support/std',
+    'frame-system/std',
+    'sp-std/std',
+    'pallet-posts/std',
+    'pallet-spaces/std',
```

Comment:
Do we really need `pallet-spaces` here? I see that you use this pallet in benchmarking to create a space. Can we put this pallet to `dev-dependencies`?

Reply:
No, we can't.